src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -173,36 +173,43 @@ To format one or more lines of code, insert three backticks ( ```` ``` ```` ) ab
   </Collapser>
 </CollapserGroup>
 
-## Customer-specific values [#value-placeholders]
+## Highlighting lines of code [#highlight-code]
 
-Traditionally, we've used `<var>` tags for customer-specific value placeholders in code snippets. For example: we'd use `<var>YOUR_ACCOUNT_ID</var>` to draw a user's attention to a place they'd need to input their account ID.
+To highlight lines of code in a code block, you can reference the specific lines of text you want highlighted in the same place where you note the syntax. Here's an example: 
 
-In November of 2022, we found out that language-specific [syntax highlighting](#synax-highlight-example) doesn't work with var tags: Using var tags in a code block disables syntax-specific coloring for that block. Going forward, for customer-specific value placeholders, avoid using var tags and instead focus on creating strong placeholder text to make it clear to customers what they should put in those places. For example, instead of using `<var>YOUR_NR_ACCOUNT_ID</var>`, we'd remove the var tags and make it a bit more descriptive, like `INSERT_YOUR_NR_ACCOUNT_ID`. 
+  ```js lineHighlight=1,3-5
+  const coolThing = {
+    key: 'HI',
+    tada: 'hello'
+  }
+  // this is an important object
+  ```
 
-That said, `<var>` tags can still be used if you think it's important to highlight something in a code snippet and you're okay losing syntax highlighting. For tips on using var tags, see [Var tags](#var-tags).
+The first line reads: `js lineHighlight=1,3-5`, which tells it to use JavaScript syntax highlighting, and tells it to highlight the first line, and the third through fifth lines. Highlighting lines requires the simultaneous use of syntax highlighting: you must indicate a language if you want to highlight lines. 
```

Comment:
@zuluecho9 readers can't see the code fence because the parser simple processes this and displays the output. How about including a screenshot of the entire snippet that shows the enclosing backticks and code fence?